Vizzuality/heco-invest
# Let 843 fe create open call form

Investors can create open calls through a form



The investor must see the content language

Respects the [design](https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=70%3A14671)

The fields validations should respect the [requirements](https://docs.google.com/spreadsheets/d/1qtP-qkm8iS6OlluiJwRvQoi6rCns9iKhRYoNzc9qkGg/edit#gid=1884416048)


The form should have 4 pages:

1. Information about the open call  
- Open call name
- Picture (optional)
- Country 
- State (optional)
- Municipality (optional)
- What is the open call about?

2. Expected impact

- Expected impact
- Select the SDG’s you expect to have impact (optional)

3. Funding information

- How much funding do you have available for this open call? ($)
- Financial instruments available
- Funding priorities
- Funding exclusions

4. How long will the open call be available?

- Calendar to select closing date

## Testing instructions

1. Sign in as a verified Investor
2. Go to /dashboard/open-calls
3. Click on the button 'create open call'. You should go to the open call creation form.
4. Fill the form and 'Submit'. The 'Save as draft is not working yet (the status doesn’t exist for open calls yet)
5. You should see the pending verification page

## Tracking

[LET-843](https://vizzuality.atlassian.net/browse/LET-843)
